src/js/components/Page/Page.js

```diff
@@ -0,0 +1,41 @@
+import React, { useContext, useMemo } from 'react';
+import { ThemeContext } from 'styled-components';
+import { Box } from '../Box';
+import { PageContext } from './PageContext';
+import { ResponsiveContext } from '../../contexts/ResponsiveContext';
+import { PagePropTypes } from './propTypes';
+import { PageContent } from '../PageContent';
+
+const Page = ({ children, plain, kind, ...rest }) => {
+  const size = useContext(ResponsiveContext);
+  const theme = useContext(ThemeContext);
+
+  const value = useMemo(
```

Comment:
very minor, but can we re-label this to be `contextValue`? I'm so used to seeing `value` in Form and *Input contexts.